### PR TITLE
Add Portuguese language & Add Cookie info translation

### PR DESCRIPTION
**Pull request by CarlosNassif**
While using this library, I found a need to use texts in Portuguese. To accomplish it, I've made some changes:

- Create the Portuguese text file with texts translated to Portuguese `pt-br.language.ts`.
- Create the type `TranslatedText`;
- Update the `NgxCookieConsentComponent` by adding dynamic translation using the `TranslatedText` type;
- Update interfaces and classes to include `TranslatedText` to the fields:
    - `CookieItem` interface:
        - name;
        - description;
        - privacyPolicyUrl.
    - `CookieDetail` interface:
        - description;
        - duration.
    - `NgxCookieConsentConfigService` class:
        - privacyPolicyUrl;
        - imprintUrl;

### CODE REVIEW

**Comment by giacomo** [comment]
@CarlosNassif Thank you for PR - i will look at it. Its a huge PR - and there are some things that i want to look closer.

File: projects/ngx-cookie-consent/src/lib/config/translated-text.type.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export type TranslatedText = {
+    [language: string]: string;
+    default: string;
```
Comment (giacomo) [inline projects/ngx-cookie-consent/src/lib/config/translated-text.type.ts:3]: default is not required here - imho - the user should define all enabled languages, at least one language

File: projects/ngx-cookie-consent/src/lib/ngx-cookie-consent.component.ts [hunk 2]
```diff
@@ -85,12 +94,15 @@ export class NgxCookieConsentComponent implements OnInit {
...
+            this.cookieForm
+                .get(category)
+                ?.get(field.key)
+                ?.setValue($event.currentTarget.checked);
         });
     }
 
     back() {
-        this.showSettingsDialog = false
+        this.showSettingsDialog = false;
```
Comment (giacomo) [inline projects/ngx-cookie-consent/src/lib/ngx-cookie-consent.component.ts:105]: good catch ;)

**Comment by CarlosNassif** [comment]
Thanks, @giacomo!
It's the first PR I write to a public lib. Please let me know if there are any adjusts I should do on code structure and logic.

**Pull request merged**